components/atoms/SearchBar.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from "prop-types";
+
+/**
+ *  component
+ */
+export function SearchBar(props) {
+  return (
+    <div className="inline-flex">
+      <form className="inline-flex py-0.5">
+        <input
```

Comment:
There needs to be a callback when the value in this input has changed. Otherwise how are we supposed to get the text in the input to the parent components. The string which is the value of the input should be passed into this callback